website/content/api-docs/secret/transform.mdx

```diff
@@ -1159,6 +1159,10 @@ This endpoint encodes the provided value using a named role.
 - `value` `(string: <required>)` –
   Specifies the value to be encoded.
 
+- `expiration` `(string: "")` - The precise expiration of the encoded value. If 
+  omitted, or no expiration is set then it shall never expire. The provided 
+  string must be a RFC3339 formatted time and date.
```

Comment:
Thanks for this! 

Could we also add the following to this

Only applicable for tokenization transformations, and will be capped by the role's `max_ttl` value if set. Both `ttl` and `expiration`  fields cannot be specified at the same time.

Reply:
@stevendpclark thanks very much for this feedback. I took your input and consolidated the text so that it read as intended. Let me know if the new proposed commit needs any extras.